ocrd_network/ocrd_network/deployer.py

```diff
@@ -515,3 +518,39 @@ def start_native_processor_server(
         stdout.close()
         stdin.close()
         return re_search(r'xyz([0-9]+)xyz', output).group(1)  # type: ignore
+
+    # TODO: No support for TCP version yet
+    def start_unix_mets_server(self, mets_path: str) -> str:
+        mets_server_url = f'/tmp/{safe_filename(mets_path)}'
+        if Path.exists(Path(mets_server_url)):
+            self.log.info(f"The mets server is already started: {mets_server_url}")
+            return mets_server_url
+        cwd = Path(mets_path).parent
+        self.log.info(f'Starting native mets server: {mets_server_url}')
+        log_path = f'{mets_server_url}.log'
+        sub_process = subprocess.Popen(
+            args=['nohup', 'ocrd', 'workspace', '--mets-server-url', f'{mets_server_url}',
+                  '-d', f'{cwd}', 'server', 'start'],
+            shell=False,
+            stdout=open(log_path, 'w'),
+            stderr=open(log_path, 'a'),
+            cwd=cwd,
+            universal_newlines=True
+        )
+        # Wait for the mets server to start
+        sleep(2)
+        self.mets_servers[mets_server_url] = sub_process.pid
+        return mets_server_url
+
+    def stop_unix_mets_server(self, mets_server_url: str) -> None:
+        self.log.info(f'Stopping native mets server: {mets_server_url}')
```

Comment:
```suggestion
        self.log.info(f'Stopping mets server: {mets_server_url}')
```

What does "native" mean here?

Reply:
It meant `UDS`. Replacing it now.